### DISCUSSION TITLE
Nyquist prompt reference.

### DISCUSSION THREAD

**Discussion by Abdullah-Ajeebi** [audacity/audacity#6700] [Q&A] [answered]
Is there a website containing every keyword of the Nyquist programing language?

**Comment by petersampsonaudacity** [comment]
@SteveDaulton  is this one you can answer?

**Comment by SteveDaulton** [comment, marked as answer]
This covers nearly all of them: https://www.cs.cmu.edu/~rbd/doc/nyquist/indx.html
